app/views/superpowers/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>List of Superpowers</h1>
+
+<ul>
```

Comment:
No need for `<ul></ul>` tags if each element is within a `<div></div>` instead of `<li></li>`! Since there aren't gonna be any list items in that ul 😉

Reply:
Removed `<ul></ul>` and left the `<div></div>`